source/plugin/Assets/GoogleMobileAds/Platforms/Unity/BannerClient.cs

```diff
@@ -146,6 +144,7 @@ public void ShowBannerView()
         {
             Debug.Log("Dummy " + MethodBase.GetCurrentMethod().Name);
             dummyAd = AdBehaviour.ShowAd(prefabAd, getRectTransform(prefabAd).anchoredPosition);
+            CreateButtonBehavior();
```

Comment:
I think I mentioned this with the callbacks but I can see how it would be confusing. All Event callbacks with OnAdOpened and OnAdLeftApplication weren't being handled properly and it was because I was calling this method CreateButtonBehaviour in the wrong spot.